bin/create.js

```diff
@@ -9,75 +8,234 @@ const { ensureDir, readdir } = require("fs-extra");
+      "None"
+    ],
+  });
+
+  switch (scFramework) {
+    case availablePackages[5].label:
+      selectedPackages.push(availablePackages[5].name);
       break;
-    case "React Native (without Expo)":
-      slug = "react-native-app-without-expo";
+    default:
       break;
-    case "Flutter":
-      slug = "flutter-app";
+  }
+
+  let { indexingProtocol } = await prompt.prompt({
+    type: "list",
+    name: "indexingProtocol",
+    message: "Create a subgraph:",
+    default: "No",
+    choices: [
+      "Yes",
+      "No"
+    ],
+  });
+
+  switch (indexingProtocol) {
+    case "Yes":
+      selectedPackages.push(availablePackages[7].name);
       break;
-    case "Angular":
-      slug = "angular-app";
     default:
       break;
   }
 
-  let { name } = await prompt.prompt({
+  let { projectName } = await prompt.prompt({
     type: "input",
-    name: "name",
-    message: "Project name : ",
+    name: "projectName",
+    message: "Project name: ",
   });
 
-  if (slug) {
+  if (selectedPackages.length > 0) {
     const pwd = process.cwd();
-    const outputDir = `${pwd}/${name}`;
+    const outputDir = `${pwd}/${projectName}`;
 
     // Ensure the output directory exists
     await ensureDir(outputDir);
     await isOutputDirectoryEmpty(outputDir);
 
     // Showing the loader
-    const spinner = loading("Celo Composer is generating your project...\n");
+    const spinner = loading(`Generation custom Celo Composer project with the following packages: ${selectedPackages}...\n`);
 
     // Shell commands to clone and trim the required directories
     shell.cd(pwd);
     shell.exec(
-      `git clone --depth 2 --filter=blob:none --sparse ${BASE_URL} ${name}`
+      `git clone --depth 2 --filter=blob:none --sparse ${BASE_URL} ${projectName}`
     );
-    shell.cd(name);
+    shell.cd(projectName);
+
+    let packageJson = {
+      "name": `${projectName}`,
+      "version": "1.0.0",
+      "description": "Custom Celo Composer project.",
+      "private": true,
+      "author": "Celo",
+      "license": "MIT",
+      "scripts": {},
+      "repository": {
+        "type": "git",
+        "url": "git+https://github.com/celo-org/celo-composer.git"
+      },
+      "bugs": {
+        "url": "https://github.com/celo-org/celo-composer/issues"
+      },
+      "homepage": "https://github.com/celo-org/celo-composer/blob/main/README.md",
+      "workspaces": {
+        "packages": [
+          "packages/*"
+        ]
+      },
+      "keywords": [
+        "celo-composer",
+        "celo"
+      ]
+    };
+
+    for (let x = 0; x < selectedPackages.length; x++) {
 
-    const { stdout, stderr, code } = shell.exec(
-      `git sparse-checkout set packages/${slug} packages/hardhat`,
-      {
-        silent: true,
+      // clone to local only the projects user wants
+      const { stdout, stderr, code } = shell.exec(
+        `git sparse-checkout add packages/${selectedPackages[x]}`, { silent: true }
+      );
+
```

Comment:
Need to add a check `selectedPackages[x] != selectedPackages[3]` (i.e. flutter-app).
@nestorbonilla

Reply:
done.